tools/gfx-unit-test/shader-cache-tests.cpp

```diff
@@ -434,7 +444,7 @@ namespace gfx_test
     //    4. #include w/ changes in the included file
     struct ShaderFileImportsShaderCache : BaseShaderCacheTest
     {
-        Slang::String importedContentsA = Slang::String(
+        String importedContentsA = String(
             R"(struct TestFunction
```

Comment:
If you insert a line break after "(" the indentation in the shader string will be consistent.

Reply:
Changed locally, will be in next push.

But honestly, I'm starting to think the shader files that are automatically written by the tests probably ought to be added to .gitignore...

Reply:
Works for me. I've deleted the old shaders locally and updated .gitignore so it doesn't pick up the renamed version of the automatically written shaders. Will be in next push